viadot/sources/vid_club.py

```diff
@@ -207,14 +225,26 @@ def get_response(
             )
         if to_date == None:
             to_date = datetime.today().strftime("%Y-%m-%d")
+        if source in ["jobs", "product", "company"]:
+            response, first_url = self.check_connection(
+                source=source,
+                from_date=from_date,
+                to_date=to_date,
+                items_per_page=items_per_page,
```

Comment:
Why region is missing here? It can be a country code or None. If it is None it shouldn't be added to 'url' but I think it can be added here - it means that second if is not necessary here.

We already handling this case in the `build_query` function:
```if source in ["jobs", "product", "company"]: lines 69-72
            url = f"{api_url}{source}?from={from_date}&to={to_date}&region={region}&limit={items_per_page}"
        elif source == "survey":
            url = f"{api_url}{source}?language=en&type=question"```